internal/cmd/agent/deployer/normalizers/jsonpatch.go

```diff
@@ -36,7 +36,7 @@ func (j JSONPatchNormalizer) Normalize(un *unstructured.Unstructured) error {
 	gvk := un.GroupVersionKind()
 	metaObj, err := meta.Accessor(un)
 	if err != nil {
-		logrus.Errorf("Failed to normalize obj with json patch, error: %v", err)
+		logrus.Debugf("Failed to normalize obj with json patch, error: %v", err)
```

Comment:
My concern here is that with debug logs disabled by default, these errors may suddenly become invisible to most users, which may result in unexpected (and now harder to troubleshoot) deployment states with bundle diffs.